src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/Constants.java

```diff
@@ -34,4 +36,6 @@ private Constants() {
 
     public static final String CONCATENATED_STRING_FORMAT = "%s,%s";
     public static final String CONCATENATED_STRING_FORMAT_NO_COMMA = "%s%s";
+
+    public static final SchemaVersion CURRENT_VERSION = SchemaVersion.VERSION_3_1;
```

Comment:
Decided that a CURRENT_VERSION constant isn't needed. We want to be explicit when referencing versions in the code.

Reply:
I guess this might be ok, as long as we don't have a 'current version' defined in the enum. @markpit thoughts?

Reply:
yes should be ok, although CURRENT_VERSION name doesn't indicate what is being versioned, CURRENT_MONGO_SCHEMA_VERSION might be better